dom.go

```diff
@@ -19,9 +19,16 @@ func Render(comp Component, container *js.Object) {
 // RenderAsBody renders the given component as the body of the page, replacing
 // whatever existing content in the page body there may be.
 func RenderAsBody(comp Component) {
-	body := js.Global.Get("document").Call("createElement", "body")
+	doc := js.Global.Get("document")
+	body := doc.Call("createElement", "body")
 	Render(comp, body)
-	js.Global.Get("document").Set("body", body)
+	if doc.Get("readyState").String() == "loading" {
```

Comment:
My minor question here is this.

Is there a non-zero possibility of a "race condition here? Imagine the page is loading when the condition on the line is evaluated. Then, it finishes loading. The next 4 lines add a listener which will never execute and return.